src/components/item.jsx

```diff
@@ -20,27 +20,41 @@ class Item extends React.Component {
   }
 
   componentDidMount  () {
+    this.setState({
+      itemId: this.props.itemId
+    });
     this.getItem();
     this.getAvailability();
     this.getReviews();
   }
 
   getItem() {
     // This will call out to Item service
+
+    // Fake data generator in case Item service unresponsive
   }
 
   getAvailability() {
     // This will call out to System Requirements service
+
+    // Fake data generator in case System Requirements service unresponsive
   }
 
   getReviews() {
     // This will call out to Reviews service
+
+    // Fake data generator in case Reviews service unresponsive
   }
 
   render() {
     return (
       <div className="item-view">
-        Item
+        ItemId: {this.state.itemId} <br/>
+        Name: {this.state.itemName} <br/>
+        Systems: {this.state.availability} <br/>
+        Good Reviews: {this.state.feedback.positiveReviews}% <br/>
+        Total Reviews: {this.state.feedback.numberOfReviews} <br/>
+        Reviews From: {this.state.feedback.reviewsFrom} <br/>
       </div>
     )
   }
```

Comment:
✓ Oh this is awesome I can see how this will come together. Also nice how you have an object in state for feedback, I swear I tried doing that once and it was giving me a good battle. Good to know!